### PR TITLE
fix:inserting failure when the values contain 'value'

### PR DESCRIPTION
**Pull request by caogaoshuai**
## What changes were proposed in this pull request

Issue Number: close #xxx 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

File: streampark-flink/streampark-flink-connector/streampark-flink-connector-base/src/main/scala/org/apache/streampark/flink/connector/failover/SinkRequest.scala [hunk 1]
```diff
@@ -30,21 +30,21 @@ case class SinkRequest(records: util.List[String], var attemptCounter: Int = 0)
   def size: Int = records.size()
 
   private[this] lazy val TABLE_REGEXP =
-    Pattern.compile("(insert\\s+into|update|delete)\\s+(.*?)(\\(|\\s+)", Pattern.CASE_INSENSITIVE)
+    Pattern.compile("^\\s*(insert\\s+into|update|delete)\\s+(.*?)(\\(|\\s+)", Pattern.CASE_INSENSITIVE)
 
   private[this] lazy val INSERT_REGEXP =
-    Pattern.compile("^(.*)\\s+(values|value)(.*)", Pattern.CASE_INSENSITIVE)
+    Pattern.compile("^\\s*(insert\\s+into)\\s+(`?([a-zA-Z]\\w*)`?.)?`?([a-zA-Z]\\w*)`?((\\s*\\((\\s*`?[a-zA-Z]\\w*`?\\s*)*\\))|(\\s+))\\s*(value|values)\\s*(\\(.*\\))\\s*;?", Pattern.CASE_INSENSITIVE)
```
Comment (zhoulii) [inline streampark-flink/streampark-flink-connector/streampark-flink-connector-base/src/main/scala/org/apache/streampark/flink/connector/failover/SinkRequest.scala]: The regex expression here is too complicated, can you make it simpler ?

Comment (caogaoshuai) [reply streampark-flink/streampark-flink-connector/streampark-flink-connector-base/src/main/scala/org/apache/streampark/flink/connector/failover/SinkRequest.scala]: Thank you for your reminder, I now use another method to fix this bug

Comment (caogaoshuai) [reply streampark-flink/streampark-flink-connector/streampark-flink-connector-base/src/main/scala/org/apache/streampark/flink/connector/failover/SinkRequest.scala]: I found the simplest solution, which only needs to add one character to the original code. The commit is "
refactor: amazing solution"

**Review by zhoulii — APPROVED** [review]
Thanks for your contribution, LGTM.

**Pull request merged**